static/js/reactions.js

```diff
@@ -7,6 +7,7 @@ import * as channel from "./channel";
 import * as emoji from "./emoji";
 import * as emoji_picker from "./emoji_picker";
 import {$t} from "./i18n";
+import {get_helpers} from "./markdown_config";
```

Comment:
As noted below, let's remove this import -- that `get_helpers` method is intended to be an internal detail of how we're sharing the Markdown processor with the mobile app.